skyportal/tests/frontend/test_followup_requests.py

```diff
@@ -97,6 +115,18 @@ def add_followup_request_using_frontend_and_verify(
     submit_button_xpath = '//form[@class="rjsf"]//button[@type="submit"]'
     driver.wait_for_xpath(submit_button_xpath)
 
+    select_box = driver.find_element_by_id(
+        "mui-component-select-followupRequestAllocationSelect"
+    )
+    select_box.click()
+
+    driver.wait_for_xpath('//li[@data-value="1"]')
+    for ii in range(1, 100):
+        allocation = driver.wait_for_xpath('//li[@data-value="%d"]' % ii)
+        if "SEDM" in allocation.text:
+            allocation.click()
+            break
```

Comment:
It think there should be a better way to select the proper allocation option in these tests than looping through every option. Probably just searching an xpath like `//li[contains(text(), "SEDM")]`.
You can then just use `driver.click_xpath(some_path, scroll_parent=True)` to find and click in one line (the `scroll_parent` tells it to scroll in the dropdown menu of options instead of the whole page when scrolling to the proper option)

Reply:
That is such an amazing trick.  Love it.